.circleci/config.yml

```diff
@@ -24,50 +24,16 @@ workflows:
               only: /^v.*/
             branches:
               ignore: /.*/
-references:
-  restore_npm_cache: &restore_npm_cache
-    restore_cache:
-      keys:
-        - npm-lock-{{ checksum "package-lock.json" }}-{{ .Environment.CIRCLE_JOB }}-{{ .Branch }}-{{ .Revision }}
-        - npm-lock-{{ checksum "package-lock.json" }}-{{ .Environment.CIRCLE_JOB }}-{{ .Branch }}
-        - npm-cache-{{ checksum "package.json" }}-{{ .Environment.CIRCLE_JOB }}-{{ .Branch }}-{{ .Revision }}
-        - npm-cache-{{ checksum "package.json" }}-{{ .Environment.CIRCLE_JOB }}-{{ .Branch }}
-  versions: &versions
-    run:
-      name: Versions
-      command: npm version
-  install: &install
-    run:
-      name: Install dependencies
-      command: npm install
-  test: &test
-    run:
-      name: Test
-      command: make ci
-  save_cache_local: &save_cache_local
-    save_cache:
-      key: npm-lock-{{ checksum "package-lock.json" }}-{{ .Environment.CIRCLE_JOB }}-{{ .Branch }}-{{ .Revision }}
-      paths:
-        - node_modules
-  save_cache_global: &save_cache_global
-    save_cache:
-      key: npm-cache-{{ checksum "package.json" }}-{{ .Environment.CIRCLE_JOB }}-{{ .Branch }}-{{ .Revision }}
-      paths:
-        - ~/.npm/_cacache
 
 version: 2
 jobs:
   node-base: &node-base
     docker:
       - image: circleci/node:latest-browsers
     steps:
-      - *versions
       - checkout
-      - *restore_npm_cache
-      - *install
-      - *test
-      - *save_cache_local
-      - *save_cache_global
+      - run: npm ci || npm i
```

Comment:
We fallback to `npm i` if the version of npm doesn't suppport `npm ci`. This is only required for node 6.